docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,279 @@
+The user has the option to setup an account on Weights & Biases to upload and view the training logs online, or on a local server to view the training logs.  Refer to the :ref:`setting-up-weights-and-biases` section below for more details.
+
+.. raw:: html
+
+   <h6>Configure Training Parameters</h6>
+You can configure the training parameters by changing the configuration yaml files or through the command line arguments. Refer to :ref:`configuring_training_parameters` for more details.
+
+.. raw:: html
+
+   <h6>Save the trained model</h6>
+The trained models will be saved in the output folder.
+
+
+
+How the training pipeline works
+===============================
+
+.. raw:: html
+
+   <h6>Overview Diagram</h6>
+
+#. Organize your data with the required format 
+#. Place them at the designated directory
+#. Personalise your training if necessary 
+#. Execute from the terminal
+
+**Component Diagram**
+
+.. image:: /assets/diagrams/C4Diagram-L4_SimplifiedOverview.png
+
+
+
+----------
+
+================
+ Getting Started
+================
+
+This guide explains how you can prepare your environment and test out the default pipeline using either TensorFlow or Pytorch.
+
+
+Install PeekingDuck Training Pipeline
+-------------------------------------
+
+.. tabs::
+
+   .. tab:: Linux
+
+      Plarform:
+         - Ubuntu 20.04 LTS / 22.04 LTS
+      Prerequisite:
+         - `Git <https://github.com/git-guides/install-git>`_
+         - `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+         - For CUDA GPU:
+            - `Latest Compatible Nvidia Display Driver for your system <https://www.nvidia.com/download/index.aspx>`_
+      
+      **CPU**
+
+      .. code-block:: bash
+         :linenos:
+
+         # Create conda environment & activate
+         conda create -n pkd-training python=3.8
+         conda activate pkd-training
+
+         # Clone PeekingDuck repository
+         git clone <TODO: PeekdingDuck repository>
+         cd PeekdingDuck
+
+         # Install required packages
+         pip install -r peekingduck/training/requirements.txt
+
+      **CUDA GPU**
+
+      .. code-block:: bash
+         :linenos:
+
+         # Create conda environment & activate
+         conda create -n pkd-training python=3.8
+         conda activate pkd-training
+
+         # Clone PeekingDuck repository
+         git clone <TODO: PeekdingDuck repository>
+         cd PeekdingDuck
+
+         # Install required packages
+         pip install -r peekingduck/training/requirements.txt
+
+         # Install cuda dependencies
+         conda install -c conda-forge cudatoolkit=11.2.2 cudnn=8.1.0
+         
+         # Initialize cuda
+         mkdir -p $CONDA_PREFIX/etc/conda/activate.d \
+         && mkdir -p $CONDA_PREFIX/etc/conda/deactivate.d \
+         && echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CONDA_PREFIX/lib/'>\
+         $CONDA_PREFIX/etc/conda/activate.d/env_vars.sh && \
+         echo 'unset LD_LIBRARY_PATH'>\
```

Comment:
Above script of 5 lines is doing a lot of things and should be split up so that each line does a single task.
Essentially, it's doing:
1. Check for existence of `activate.d` / `deactivate.d` and create it if not found
2. Add the LD_LIBRARY_PATH into `env_vars.sh`
3. Print a message

Lumping everything into one makes it hard to read and maintain.

Reply:
Noted. The initial consideration was for the user to execute the commands easily with a single command, but agree with the readability. Will separate them out.